pkg/mkpersist/mkpersist.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+#
+# Copyright (c) 2018 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+PERSISTDIR=/var/persist
+mkdir -p $PERSISTDIR
+chmod 700 $PERSISTDIR
+P3_FS_TYPE="ext3"
+FSCK_FAILED=0
+#For systems with ext3 filesystem, try not to change to ext4, since it will brick
+#the device when falling back to old images expecting P3 to be ext3. Migrate to ext4
+#when we do usb install, this way the transition is more controlled.
+if P3=$(zboot partdev P3) && [ -n "$P3" ]; then
```

Comment:
This seems to be the only dependency on zboot/gpttools here so lets just replace it with a direct call to: findfs PARTLABEL=P3

Reply:
will do.